front/components/RedirectOnLoad/RedirectOnLoad.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { Redirect } from 'react-router-dom';
+
+import { AppDataContext } from '../..';
+
+// RedirectOnLoad assesses the initial state of the application using appData and determines the proper
+// route to load in the Router
+export const RedirectOnLoad = () => {
+  return (
+    <AppDataContext.Consumer>
+      {appData => {
+        const { state, video } = appData;
+
+        // Get LTI errors out of the way
+        if (state === 'error') {
+          return <Redirect push to="/errors/lti" />;
```

Comment:
may not be DRY as I suppose that you refer to "/errors/lti" somewhere else as a state?